Makefile

```diff
@@ -84,33 +84,43 @@ ${PREFIX}/bin/static/notary:
 	@go build -tags "${NOTARY_BUILDTAGS} netgo" -o $@ ${GO_LDFLAGS_STATIC} ./cmd/notary
 endif
 
+ifeq (, $(shell which staticcheck))
+STATICCHECK_BIN := $(GOBIN)/staticcheck
+$(STATICCHECK_BIN):
+	@echo "+ $@"
+	GOFLAGS="-mod=mod" go install honnef.co/go/tools/cmd/staticcheck@latest
```

Comment:
`GOFLAGS="-mod=mod"` is necessary here because, most probably, circelCI is set with `-mod=vendor`